dev/drivers/scripts/global_ens/prep/jevs_global_ens_naefs_atmos_prep.sh

```diff
@@ -12,45 +10,33 @@
 
 set -x
 export OMP_NUM_THREADS=1
-export evs_ver=v1.0
-export HOMEevs=/lfs/h2/emc/vpppg/noscrub/${USER}/EVS
+export HOMEevs=/lfs/h2/emc/vpppg/noscrub/$USER/gitworkspace/EVS
+#export HOMEevs=/lfs/h2/emc/vpppg/noscrub/${USER}/EVS
 source $HOMEevs/versions/run.ver
 
-module reset
-module use /apps/ops/para/libs/modulefiles/compiler/intel/${intel_ver}
-export HPC_OPT=/apps/ops/para/libs
-module use /apps/dev/modulefiles/
-module load ve/evs/${ve_evs_ver}
-module load gsl/${gsl_ver}
-module load met/${met_ver}
-module load metplus/${metplus_ver}
-module load cray-pals/${craypals_ver}
-module load prod_envir/${prod_envir_ver}
-module load prod_util/${prod_util_ver}
-module load cfp/${cfp_ver}
-module load libjpeg/${libjpeg_ver}
-module load libpng/${libpng_ver}
-module load zlib/${zlib_ver}
-module load jasper/${jasper_ver}
-module load udunits/${udunits_ver}
-module load grib_util/${grib_util_ver}
-module load wgrib2/${wgrib2_ver}
-export MET_bin_exec=bin
-
-module list
-
 export NET=evs
 export RUN=atmos
 export STEP=prep
-export COMPONENT=naefs
+#export COMPONENT=naefs
+export COMPONENT=global_ens
 export VERIF_CASE=grid2grid
-export MODELNAME=gefs
+#export MODELNAME=gefs
+export MODELNAME=naefs
+
+
+module reset
+
+source $HOMEevs/modulefiles/global_ens/global_ens_${STEP}.sh
```

Comment:
Modify to use $COMPONENT in place of global_ens.

Reply:
YOu can still take this one further ... use $COMPONENT for all global_ens instances.
From
`source $HOMEevs/modulefiles/${COMPONENT}/global_ens_${STEP}.sh`
 to
`source $HOMEevs/modulefiles/${COMPONENT}/${COMPONENT}_${STEP}.sh`.

Reply:
Done!